### PR TITLE
Add Captive Audience to the book list

### PR DESCRIPTION
**Pull request by dhcole**
Adds a record for a great book on the history of broadband and the telecom industry with a focus on municipal broadband policy.

### CODE REVIEW

File: _data/books.yml [hunk 1]
```diff
@@ -76,6 +76,10 @@ Technology policy:
     asin: B0015GWX0S
     tldr: Collaborators and non-market actors (wikipedia, open source) are king in the approaching "industrial information economy"
 
+  - title: "Captive Audience: The Telecom Industry and Monopoly Power in the New Gilded Age"
+    asin: 0300153139
```
Comment (mibo420) [inline _data/books.yml]: 0300153139

File: _data/books.yml [hunk 2]
```diff
@@ -76,6 +76,10 @@ Technology policy:
...
+    tldr: '"Ten years ago, the United States stood at the forefront of the Internet revolution. With some of the fastest speeds and lowest prices in the world for high-speed Internet access, the nation was poised to be the global leader in the new knowledge-based economy. Today that global competitive advantage has all but vanished because of a series of government decisions and resulting monopolies that have allowed dozens of countries … to pass us in both speed and price of broadband."'
+
 Government and organizing:
 
   - title: "Bureaucracy: What Government Agencies Do And Why They Do It"
```
Comment (mibo420) [inline _data/books.yml]: Bureaucracies :What Government Agencies do and why they do it.

**Comment by mibo420** [comment]
Thank You for your help in my lessons this morning.Was very kind of you,and
i appreciate being a Google Customer very much. Thanks for everything, and
HaveFun! At Tumblr @WickedWishes & Twitter @WishesWicked. MITCHELL L
Boedecker https://mibo420@tumbler.com  This is wrong in my net email
settings,so I can't receive mail,as server doesn't recognize me yet.Thanks
again,and Have Fun!

WickedWisher

**Comment by mibo420** [comment]
Thanks Ben.Hope all works out for future endeavors. Have Fun! @
WickedWishes Thanks alot. MITCHELL L Boedecker  https://mibo420@tumblr.com
 Twitter @WishesWicked

WickedWisher

File: _data/books.yml [hunk 3]
```diff
@@ -76,6 +76,10 @@ Technology policy:
     asin: B0015GWX0S
```
Comment (mibo420) [inline _data/books.yml]: https://mibo420.tumbler.net @ wickedwishes/tumbler &wisheswicked/Twitter &@ gmail accounts do recognize me, and HaveFun! Thanks Jessica Have Fun!

**Comment by benbalter** [comment]
Thanks. For now, keeping it a list of books _I've_ read, but will add it to my Amazon wish list -- looks interesting! Thanks for the PR! 😄

**Pull request closed without merging**